src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -35,11 +44,37 @@ export const FlowChartWrapper = ({
   onToggleModularPipelineExpanded,
   onUpdateActivePipeline,
   pipelines,
-  reload,
   sidebarVisible,
+  metadataVisible,
 }) => {
+  const history = useHistory();
+
+  // Reload state is to ensure it will call redirectLocation
+  // only when the page is reloaded.
+  const [reload, setReload] = useState(false);
+
+  const [counter, setCounter] = React.useState(60);
+  const [goBackToExperimentTracking, setGoBackToExperimentTracking] =
+    useState(false);
+
+  useEffect(() => {
+    setReload(true);
+
+    const storage = window.localStorage.getItem('kedro-viz-link-to-flowchart');
```

Comment:
I am a little confused by how this storage works. Maybe we can go thru this on call